### PR TITLE
Improved error texts

### PR DESCRIPTION
**Pull request by arnoutt**
As preparation for the new error page designs.

### CODE REVIEW

File: languages/messages.en.php [hunk 1]
```diff
@@ -179,75 +179,54 @@
 <ul><li>restart your browser</li>
 <li>login again</li>
 <li>share your information</li></ul>',
-    'error_no_idps'                 => 'Error - No Identity Providers found',
-    'error_no_idps_desc'            => '<p>
-The service you\'re trying to reach (&lsquo;Service Provider&rsquo;) is not accessible through %suiteName%.<br /><br />
-    </p>',
+    'error_no_idps'                 => 'Error - No %organisationNounPlural% found',
+    'error_no_idps_desc'            => 'Logging into this service is not possible via %suiteName%. The service is not connected to any institutions.',
```
Comment (thijskh) [inline languages/messages.en.php]: Also use `%organisationNounPlural%` here instead of institutions

**Comment by thijskh** [comment]
@pablothedude Arnout's change drops a number of '<p>' tags. Is that a problem you think? It does look better to me now not having that HTML in there...

**Comment by pablothedude** [comment]
> @pablothedude Arnout's change drops a number of `<p>` tags. Is that a problem you think? It does look better to me now not having that HTML in there...

@thijskh it shouldn't be an issue because it would only be used for styling and if it looks good to you I'm fine with that. I don;t now if you've seen the changes yourself else it would be possible to generate some screen shots in order to validate the changes.

**Comment by thijskh** [comment]
@pablothedude This can be merged but also needs to be merged to 5.11, can you do that?

**Comment by pablothedude** [comment]
Done

**Pull request merged**